chart/templates/gateway-ingress-gateway.yaml

```diff
@@ -27,9 +27,18 @@ spec:
       name: https
       protocol: HTTPS
     tls:
+{{ if eq .Values.gateway.qwac.enabled -}}
+      mode: SIMPLE
+      credentialName: gateway-ingress-certificate-for-signin-sealed-secret
+{{ else -}}
       mode: SIMPLE
       serverCertificate: sds
       privateKey: sds
       credentialName: {{ .Release.Name }}-gateway-ingress-certificate
-    hosts:
-    - {{ include "gateway.host" . }}
+{{ end -}}
+hosts:
+{{ if eq .Values.gateway.qwac.enabled -}}
+  - {{ include "gateway.host" . }}
+{{ else -}}
+  - {{ include "gateway.host.govuk" . }}
+{{ end -}}
```

Comment:
I'd prefer one if/else block per case that contains the entire `tls` and `hosts` config, I'd find it easier to compare

Reply:
I'd still prefer a bit of repetition and two distinct blocks i.e.
````
    {{- if eq .Values.global.qwacCertificate.enabled -}}
    tls:
      mode: SIMPLE
      credentialName: gateway-ingress-certificate-for-signin-sealed-secret
    hosts:
    - {{ include "gateway.host.govuk" . }}
    {{- else -}}
    tls:
      mode: SIMPLE
      serverCertificate: sds
      privateKey: sds
      credentialName: {{ .Release.Name }}-gateway-ingress-certificate
    hosts:
    - {{ include "gateway.host" . }}
    {{- end -}}
````
Also note the [chomp-left dash on the flow control statements](https://helm.sh/docs/chart_template_guide/control_structures/#controlling-whitespace)

Reply:
That's what I did isn't it?  And I left the chomp left out as there's nothing to chomp but I'll add them in if you want.

Reply:
I see, you want TLS inside too. Fair enough.